src/main/scala/cognite/spark/v1/FileContentRelation.scala

```diff
@@ -34,7 +35,10 @@ class FileContentRelation(config: RelationConfig, fileExternalId: String, inferS
     with Serializable
     with WithSizeLimit {
 
-  override val sizeLimit: Long = 5 * FileUtils.ONE_GB
+  override val fileSizeLimitBytes: Long = 5 * FileUtils.ONE_GB
+  //We enforce a 2.5MB limit per line. In jvm each character takes 2bytes so we divide by two to get the limit in characters.
+  private val lineSizeLimitBytes: Int = (2.5 * FileUtils.ONE_MB).toInt
```

Comment:
would make sense to put that into trait & parameter too? If anything it can help writing a unit test for this part

Reply:
ah, neither is a parameter